template/shared/client/app.js

```diff
@@ -13,41 +13,47 @@ var domReady = require('domready');
 
 module.exports = {
     // this is the the whole app initter
-    blastoff: function () {
-        var self = window.app = this;
-
-        // create our global 'me' object and an empty collection for our people models.
-        window.me = new Me();
-        this.people = new People();
-
-        // init our URL handlers and the history tracker
-        this.router = new Router();
+    blastoff: function() {
+        app.extend({
+
+            // create the global 'me' object and an empty collection for our people models
+            me: new Me(),
+            users: new People(),
+
+            // init our URL handlers and the history tracker
+            router: new Router(),
+
+            // This is how you navigate around the app.
+            // this gets called by a global click handler that handles
+            // all the <a> tags in the app.
+            // it expects a url without a leading slash.
+            // for example: "costello/settings".
+            navigate: function(page) {
```

Comment:
@mikehedman I think there's some more we can do here while still maintaining flexibility.

I'm going to tweak some of this a bit and also go add some tests and such. Thanks for doing the initial conversion @fyockm. More thoughts incoming...